tests/Main.hs

```diff
@@ -23,44 +24,28 @@ import qualified TestMake
 import qualified TestUtils
 import qualified TestGraph
 
-import System.IO (hSetEncoding, stdout, stderr, utf8)
+import System.IO (hSetBuffering, stdout, BufferMode(..))
 
 main :: IO ()
 main = do
-  hSetEncoding stdout utf8
-  hSetEncoding stderr utf8
+  hSetBuffering stdout LineBuffering
```

Comment:
I was finding that the output was being buffered in larger blocks, perhaps because of how stack runs these tests. This line fixed the issue.

Reply:
I tested and it seems this line is no longer necessary with the new bleeding-edge Hspec.